panther_analysis_tool/main.py

```diff
@@ -911,6 +912,12 @@ def setup_run_tests(  # pylint: disable=too-many-locals,too-many-arguments
             filters=analysis_spec.get(BACKEND_FILTERS_ANALYSIS_SPEC_KEY) or None,
         )
 
+        if is_correlation_rule(analysis_spec):
+            logging.warning(
+                "Skipping Correlation Rule '%s', testing not supported", analysis_spec.get("RuleID")
+            )
```

Comment:
I feel like we don't need a message here. If testing isn't supported, it is verbose to state that every time tests are run. We can make this a debug instead so that info is still available

Reply:
I think we should if they are normally something that is tested, but this is a special case where are not tested. But testing for correlation rules isn't even a part of our product (so far). You don't think that would clutter up user's output when running tests?